### ISSUE TITLE
Can't move shortcuts in applications or start menu windows 1909

### ISSUE THREAD

**Issue by glowingwire** [Open-Shell/Open-Shell-Menu#268]
I can't move or drag any menu items is the start menu like I used to be able to ... I'm now using 4.4.142 and recently had my computer update to Windows1909.

It's all I can do from continuously cursing Microsoft.

**Describe the bug**
Basically the bug is Microsoft is making it so we can't use their product. 
Microsoft
Quick Launches also disappears on updates...

**To Reproduce**
1. Go to Microsoft Headquarters
2. Click on Balmer
3. Scroll down to Current CEO
4. See That windows isn't tested at all in any way that matters, and the computer is unusable, even when steps were taken to make it work.

**Expected behavior**

**Screenshots**

**Version:**
 - Open-Shell: [e.g. 4.4.131]
 - OS: [e.g. Windows 10 1903]

**Additional context**

**Comment by blackcrack** [comment]
@ge0rdi 
@coddec 
@Ibuprophen 

can anyone this close, there is something wrong i guess..

**Comment by Ibuprophen** [comment]
I'm not exactly clear as to the question being asked and I'd hate to assume with an unhelpful response... 

~Ibuprophen

**Comment by glowingwire** [comment]
I'm sorry for the cheeky jab at Microsoft... 
I can't move any menu items in windows 1909 and the latest nightly build.

I can move menu items on windows 1803 (I have no idea why my computer won't update... I think I refused to take the sd card out...) and.. Classic Shell 4.3.1  (I still haven't updated this computer to Open Shell)
It's kind of important to be able to organize the start menu items and programs, and I used to keep them all well sorted.

If I really get involved, I'll try to do a categorization system like windows does and have a service that arranges all of my programs according to category, and when the applications get updated doesn't make a duplicate entry (I get behind on organizing my start menu, and I'm currently very far behind, and now I can't correct the issue)


On my 1909 computer when I try to drag the file a windows move file dialog shows up, and it won't actually move anything. 
I suspect that we will have to grant OpenShell more permissions... maybe run as administrator or something stupid like that  as they do tend to lock down security as versions increase... causing more problems

I may have seen this listed in the release notes that it was a problem.
